packages/x-data-grid/src/components/containers/GridToolbarContainer.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import { styled, SxProps, Theme } from '@mui/system';
-import { unstable_composeClasses as composeClasses } from '@mui/utils';
+import composeClasses from '@mui/utils/composeClasses';
```

Comment:
Nice, looks like we can have the flattening of imports as it's own PR.

Reply:
It's one of the goal of #13401 but it's blocked because we use the `unstable_` part which is really not nice

Reply:
Ah right, I have missed this. We can't make this change.

Reply:
By the way, it's a small argument in favor of named import:
You can do:

```ts
import { unstable_composeClasses } from '@mui/utils/composeClasses'
```

With default import we'd need to do this to keep the info:

```ts
import { unstable_composeClasses } from '@mui/utils/unstable_composeClasses'
```

Reply:
Now, I guess I that we can do 
```suggestion
import unstable_composeClasses from '@mui/utils/composeClasses';
```
and call it a day.

Reply:
That's a fragile convention :laughing:

Reply:
I think we need a way to share some private stuff, especially as our amount of component increases.

But yeah some (most ?) of the APIs in `@mui/utils` could probably be public and stable by now.

Reply:
> But yeah some (most ?) of the APIs in @mui/utils could probably be public and stable by now.

Agree, these are here for years already.